src/adapters/lokijs/WorkerBridge.js

```diff
@@ -25,7 +25,8 @@ class WorkerBridge {
   _pendingRequests: WorkerActions = []
 
   _createWorker(): Worker {
-    const worker: Worker = new LokiWorker()
+    const WorkerClass = (LokiWorker: any)
+    const worker: Worker = new WorkerClass()
```

Comment:
Why not `const worker: Worker = (new LokiWorker(): any)`?

Reply:
the problem is with `LokiWorker`, because Flow doesn't understand it's always a Class, and thinks it might be `any` (and therefore `new x()` is illegal)

Reply:
Hmm are you sure? I just changed it locally and `ci:check` shows no errors.

Reply:
I'll try again ;)